### ISSUE TITLE
Add a few missing __init__ entries to dist3.py

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8543]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8543|
|Type|enhancement|
|Created|2016-06-30 08:58:17Z|
|Branch|https://github.com/twisted/twisted/tree/8543-rodrigc-missinginit|

The following __init__ entries are missing in dist3.py:

* twisted.persisted.test.__init__
* twisted.positioning.test.__init__
* twisted.python.test.__init__
* twisted.web._auth.__init__

They are needed to run more tests on Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8543 8543
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8543_rodrigc_missinginit 8543-rodrigc-missinginit
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467277097546542 1467277097546542
changetime__1467279173096887 1467279173096887
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/276

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Look good. Thanks. Please merge!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 594adb889cb95868fcba378f1d69f213ea187fd8
```
#!CommitTicketReference repository="" revision="594adb889cb95868fcba378f1d69f213ea187fd8"
Merge 8543-rodrigc-missinginit: Add 4 missing __init__ entries to dist3.py to enable more tests to run on Python 3

Author: rodrigc
Reviewer: adiroiban
Fixes: #8543
```
